lib/routes/requests.js

```diff
@@ -0,0 +1,97 @@
+var Boom = require('boom')
+
+/* NOTES:
+  * What Auth do we use against CouchDB to get/set user doc? (user not logged in)
+  * What url do we send to the user? (how do configure url)
+  * Could we use promises? I think it will clean the code up nicely
```

Comment:
We usually stick to callbacks in node code as it feels more natural for most node developers, but this is work in progress code, and especially as this is your first contribution, I'd say use whatever you are most comfortable with, we can still change things later once it's working, no big deal at all :)